test/circuit_breaker_test.rb

```diff
@@ -117,7 +120,7 @@ def test_open_close_open_cycle
       open_circuit!(resource)
       assert_circuit_opened(resource)
 
-      Timecop.travel(half_open_future_travel.call) do
+      Timecop.travel(resource.circuit_breaker.error_timeout + 1) do
```

Comment:
This isn't equivalent to the previous code right?

`Time.now` was returning two different values for the two `half_open_future_travel.call` calls.

Reply:
TIL!